app/src/main/java/com/android/sample/model/recipe/Recipe.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.sample.model.recipe
+
+data class Recipe(
+    val idMeal: Long,
+    val strMeal: String,
+    val strCategory: String,
+    val strArea: String,
+    val strInstructions: String,
+    val strMealThumb: String,
```

Comment:
For the thumbnail can you change the name to strMealThumbUrl, because I'm guessing that's what the string will be.